bson/src/test/unit/org/bson/types/ObjectIdTest.java

```diff
@@ -146,33 +147,33 @@ public void testToHexString() {
                 new ObjectId(Integer.MAX_VALUE, Short.MAX_VALUE, Short.MAX_VALUE, Short.MAX_VALUE).toHexString());
     }
 
-    private void checkDates(final String dateInTime, final ObjectId oid) {
-        SimpleDateFormat dateFormat = new SimpleDateFormat("dd-MM-yyyy HH:mm:ssZ");
-        try {
-            Date d = dateFormat.parse(dateInTime);
-            assertEquals(oid.getDate(), d);
-        } catch (Exception e) {
-        }
+    private Date getDate(String s) throws ParseException {
+        SimpleDateFormat dateFormat = new SimpleDateFormat("dd-MMM-yyyy HH:mm:ss Z");
```

Comment:
Nit: generally we avoid extraneous local variables, so this and the ones below can be inlined (there's an automatic refactoring that will do it. :) )

Reply:
I have refactored the code for this method and the ones below.